Automattic/wp-calypso
# Update support links to subdirectory

#### Changes proposed in this Pull Request

* This change updates all support document links from subdomain format to subdirectory format. This is along the works being done here p7DVsv-8ak-p2

#### Testing instructions



* Visit support links throughout Calypso (it may be hard to go through all changes; so a few is fine)
* Ensure that they take you to https://wordpress.com/support (EN) or https://wordpress.com/de/support (locale) format. 

They must not take you to https://en.support.wordpress.com, https://support.wordpress.com or https://de.support.wordpress.com or similar formats. Technically these will redirect, but we prefer to show the actual link within Calypso.

#### Notes

I used the regex in this order:

- `https://en.support.wordpress.com` replaces with `https://wordpress.com/support`
- `https://(.*).support.wordpress.com` replaces with `https://wordpress.com/$1/support`
- `https://support.wordpress.com` replaces with `https://wordpress.com/support`
